README.md

```diff
@@ -32,6 +32,13 @@ Environment variables:
     docker will remove the image and all intermediate parent images.
   * `ARTIFACTS_DIR`  
     Name of the directory into which logs and similar debug output is placed.
+  * `TEST_IN_TEAMCITY`  
```

Comment:
for another PR since this is only about docs, what about deriving `TEST_IN_TEAMCITY` from the presence of `TEAMCITY_API_TOKEN`?

Reply:
So `TEST_IN_TEAMCITY` does also alter the output format. The API token is only required for downloading artifacts... They are tightly coupled indeed and I cannot think of a use case where you'd want one but not the other...